src/core/trulens/core/tru.py

```diff
@@ -881,51 +873,15 @@ def runloop():
             )
             print(
                 f"Will rerun running feedbacks after "
-                f"{humanize_seconds(self.RETRY_RUNNING_SECONDS)}."
+                f"{format_seconds(self.RETRY_RUNNING_SECONDS)}."
             )
             print(
                 f"Will rerun failed feedbacks after "
-                f"{humanize_seconds(self.RETRY_FAILED_SECONDS)}."
+                f"{format_seconds(self.RETRY_FAILED_SECONDS)}."
             )
 
             total = 0
```

Comment:
What's the reason for removing the tqdm stats?

Reply:
Was working on stripping out non-essential dependencies from trulens-core but can bring back the stats if needed

Reply:
May be file another JIRA for the stats.